compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/MoveCompanionObjectFieldsLowering.kt

```diff
@@ -83,9 +84,15 @@ private class MoveCompanionObjectFieldsLowering(val context: CommonBackendContex
         companion.declarations.removeAll { it is IrAnonymousInitializer }
     }
 
-    private fun IrClass.allFieldsAreJvmField() =
-        declarations.filterIsInstance<IrProperty>()
-            .mapNotNull { it.backingField }.all { it.hasAnnotation(JVM_FIELD_ANNOTATION_FQ_NAME) }
+    private fun shouldExposeProperty(irClass: IrClass, irProperty: IrProperty): Boolean {
```

Comment:
What does "exposing" the property mean? I suggest renaming it (`shouldMoveProperty`?) since "expose" isn't used anywhere else.

Reply:
It's used in:
https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#static-fields

meaning that exposing a Kotlin property as a Java field.

Reply:
I'll use shouldMoveProperty and document the link in comments.